config/kernelci.conf

```diff
@@ -24,6 +24,8 @@ plan: check-describe
 [kcidb]
 
 [test_report]
+email_hostname: mail.collabora.co.uk
+email_hostport: 587
```

Comment:
Can we keep using collabora's server here?

Reply:
I think we shouldn't leave this address here. I'd rather put some example values here. @gctucker , what do you recon?

Reply:
I guess one issue is that `kernelci.conf` is under source control, as it's in principle the correct file for local settings but we can't put any email config by default as every instance will be using a different one.  So in practice, having environment variables like we're doing with the port numbers for `kernelci-api` would probably work best.  We'll need that at least for SMTP passwords I guess.

Reply:
Later on, when we have proper YAML config for emails, we would have an `--email-config` parameter and sections in the config file for each email server the user wants to set up.  Then we could maybe add some `sendmail` example with a localhost setting.